.github/workflows/gcp-deploy.yaml

```diff
@@ -19,14 +19,36 @@ env:
   BACKEND_PORT: 4000
   HASURA_PORT: 8080
   FRONTEND_PORT: 3000
+  HASURA_SECRET: metagame_secret
 
 jobs:
+  start-deployment:
+    name: Generate Deployment Start Message
+    runs-on: ubuntu-latest
+
+    outputs:
+      deployment_id: ${{steps.create-message.outputs.deployment_id}}
+
+    steps:
+      - name: Start Deployment
+        uses: bobheadxi/deployments@v0.6.2
+        id: create-message
+        with:
+          step: start
+          token: ${{github.token}}
+          env: test
+          desc: |
+            "Test instance deployment for PR #${{github.event.number}} \
+            of ${{github.event.pull_request.head.label}} \
+            by ${{github.event.pull_request.user.login}}"
+          ref: ${{github.event.pull_request.head.sha}}
+
   cancel-previous:
     name: Cancel Existing Workflows
     runs-on: ubuntu-latest
 
     steps:
-      - name: Cancel Existing Runs
+      - name: Cancel Running Actions
```

Comment:
the previous "cancel existing runs" might actually be more accurate since this just cancels currently running runs of this action for this pr